### ISSUE TITLE
Image segmentation using custom neural network model with Tensorflow

### ISSUE THREAD

**Issue by gusqgm** [fractal-analytics-platform/fractal-tasks-core#9]
Currently two different network architectures have been used for object segmentation: [Cellpose](https://github.com/mouseland/cellpose) and [RDCNet](https://github.com/fmi-basel/RDCNet). As a minimal working example, we should be able to use a pre-trained model from each of these architectures and use it for prediction of label maps. Important to note that Cellpose is using pytorch, whereas RDCNet utilizes Tensorflow.

Cellpose should work by installing it from the repository above. For RDCNet, we have usually used it via [ISIT](https://github.com/fmi-basel/isit).

The input is the reading of the zarr file in such a way that the images match the images used for training and applying the prediction model on them. The output are object labels which are defined as regions which contain information of where they should be located within the field grid. Each region comprises of a segmented object plus a certain bounding box surrounding it.

Currently the models mostly used with RDCNet rely on object segmentation of MIP projections of each well/overview.

**Comment by jluethi** [comment]
This has been covered in the (private) RDCNet task at FMI now
